### PR TITLE
Backport JSONObject.wrap in order to support Jellybean

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

File: shell/platform/android/io/flutter/plugin/common/JSONUtil.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package io.flutter.plugin.common;
+
+import java.lang.reflect.Array;
+import java.util.Collection;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class JSONUtil {
+    /**
```
Comment (mravn-google) [inline shell/platform/android/io/flutter/plugin/common/JSONUtil.java]: You may want to add a private constructor to make this class uninstantiable.

**Comment by abarth** [comment]
LGTM

File: shell/platform/android/io/flutter/plugin/common/JSONUtil.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package io.flutter.plugin.common;
+
+import java.lang.reflect.Array;
+import java.util.Collection;
+import java.util.Map;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class JSONUtil {
```
Comment (mravn-google) [inline shell/platform/android/io/flutter/plugin/common/JSONUtil.java]: This class could be package scope.

Comment (jason-simmons) [inline shell/platform/android/io/flutter/plugin/common/JSONUtil.java]: The class is also used by the plugin.editing package

File: shell/platform/android/io/flutter/plugin/common/JSONUtil.java (hunk 1 above)
Comment (jason-simmons) [inline shell/platform/android/io/flutter/plugin/common/JSONUtil.java]: done

**Pull request merged**